### ISSUE TITLE
Error: project.yaml created for different versions

### ISSUE THREAD

**Issue by keyurshah** [putyourlightson/craft-blitz#117]
I'm receiving the error below when trying to apply a project update after upgrading blitz. 

```
Your project.yaml file was created for different versions of Craft and/or plugins than what’s currently installed.

Blitz currently installed has the schema version of 2.1.0 (expected 2.0.0)
Try running composer install from your terminal to resolve.
```

I had issues with a previous upgrade and I thought I fixed it by enabling AllowAdminChanges manually on staging as discussed here #114 

---

So I attemped to manually fix the error, using the following steps in case it can help anyone. This seems to have fixed the issue for now.

I went into project.yaml and changed it manually from:

```
  blitz:
    edition: standard
    enabled: true
    schemaVersion: 2.0.0
```

to

```
  blitz:
    edition: standard
    enabled: true
    schemaVersion: 2.1.0
```

then I redeployed. It prompted me for which project.yaml file to use, and I said use the one from the file, instead of the one already loaded.

Then it seemed to work once I applied the changes. 

Just wanted to give a heads up as I think those 2 issues might be linked. 

Thanks.

**Comment by putyourlightson** [comment]
Thanks for posting this. I think you will find however, that manually editing `project.yaml` is likely to create further issues down the road when syncing environments. The issue is in fact to do with Craft and not Blitz, but I believe that updating plugins should only really happen on dev environments where `allowAdminChanges` is set to `true`. Once you deploy to your other environments, `project.yaml` should sync the changes for you.

Reference: https://github.com/craftcms/cms/issues/3653

**Comment by keyurshah** [comment]
On my staging server I have `allowAdminChanges` set to `false` normally, but I had to change to `true` to fix #114. I then forgot to change it back when I deployed again, so maybe that caused the issue.

At this point, I will try deleting Blitz and reinstall to try to get everything back in alignment. hopefully that will help for future updates
